### ISSUE TITLE
Any chance of this getting on Maven?

### ISSUE THREAD

**Issue by lare96** [maxaf/numerato#2]
Trying to use it for a Java and Scala project. Since the majority of the project is in Java, Maven is used to manage dependencies

**Comment by maxaf** [comment]
Pushing to central is definitely something I want to do for is and other projects. I'll work that out soon (just got back from vacation); for the time being, would you be able to add my artifact repository to your POM? There are [instructions for SBT](https://github.com/maxaf/numerato#using-numerato-in-your-project) - let me know if it would be helpful to have Maven-specific POM snippets added to the README as well.

**Comment by lare96** [comment]
Of course, no worries. It would definitely be helpful to have Maven-specific POM snippets added though!
